Website/src/rendering/src/assets/css/layout/information-section.css

```diff
@@ -21,7 +21,15 @@
       @apply md:col-span-3;
 
       .eyebrow {
-        @apply uppercase;
+        @apply font-medium uppercase bg-yellow inline-block py-2 px-6;
+
+        margin-top: 0px !important;
```

Comment:
We should avoid `!important`. Why is it needed?

Reply:
without this tailwind utilities is adding margin top on the second element that I cannot remove. 
looks odd
![image](https://user-images.githubusercontent.com/82103130/134973819-5e9bc6ef-d41d-442d-b2ce-4b3e6abcbfca.png)

Reply:
This is caused by `description-col` `space-y-5`.
Each direct child of the description-col is spaces by 5. The solution is to create an eyebrow container to put all eyebrow cards in it.

Reply:
right, totally forgot. did that for session but forgot to do it for speakers.